src/video/canvas.cpp

```diff
@@ -331,6 +332,26 @@ Canvas::draw_triangle(const Vector& pos1, const Vector& pos2, const Vector& pos3
   m_requests.push_back(request);
 }
 
+void
+Canvas::draw_hexagon(const Vector& pos, float radius, const Color &color,
+  int layer)
+{
+  float radius2 = radius * sqrtf(0.8f);
+  float x_off_small = radius * sqrtf(0.2f);
+  std::array<Vector, 6> offsets{
```

Comment:
I believe this can be marked `const`.

Reply:
I think `const` would be redundant here since `offsets` is only a local variable.

Reply:
Still allows for const-correctness, though. Do it if you wish to.

Reply:
I'm omitting it for brevity.